### PR TITLE
chore(release): Publish 10-08-2020

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Merging released code into master (version bumps and changelogs)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_analytics_pinpoint/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 0.0.1-dev.3 (2020-10-08)
```
Comment (jamesonwilliams) [inline packages/amplify_analytics_pinpoint/CHANGELOG.md:1]: This is really driving the point hope that it's a pre-release 😀.

Anything below 1.0.0 is acceptable. I'd drop the -dev business and cut a minor.

0.1.0?

For reference, the initial iOS and Android releases were 0.9, signifying that they were working towards 1.0.0, but were not quite there yet. We ended up having 0.10.x etc., before we pinned stable APIs in 1.0.0.

Comment (Ashish-Nanda) [reply packages/amplify_analytics_pinpoint/CHANGELOG.md:1]: Agree. We can change it to 0.x.x when we announce our release candidate with DataStore and API along with other improvements.

File: packages/amplify_auth_cognito/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+* fix(amplify_auth_cognito): throw error in iOS for unconfirmed user signin
+* fix(amplify_auth_cognito): returns AuthError instead of missing method error
+* feat(amplify_auth_cognito): iOS unit tests
+* fix(amplify_auth_cognito): fix example app Podfile
+* feat(amplify_auth_cognito): integration test initial pass
+* fix(amplify_auth_cognito): truncates android resendsignup response
+* Update example podspecs
+* fix(amplify_auth_cognito): event stream destruction
+* chore: unify gradle versions across repo
```
Comment (Ashish-Nanda) [inline packages/amplify_auth_cognito/CHANGELOG.md:13]: Nit: There are some features/chores under the bug fix heading.

**Review by Ashish-Nanda — APPROVED** [review]
LGTM.
Left a minor comment about updating the `amplify_auth_cognito` CHANGELOG to have features under a different heading from bug fixes.

File: packages/amplify_auth_cognito/CHANGELOG.md (hunk 2 above)
Comment (haverchuck) [reply packages/amplify_auth_cognito/CHANGELOG.md:13]: @Ashish-Nanda True... but the only "features" are unit tests.

File: packages/amplify_analytics_pinpoint/CHANGELOG.md (hunk 1 above)
Comment (haverchuck) [reply packages/amplify_analytics_pinpoint/CHANGELOG.md:1]: I don't think there is anything wrong with driving a point home when it comes to OSS... but if we want to drop the -dev suffix, that's fine by me.

Comment (Amplifiyer) [reply packages/amplify_analytics_pinpoint/CHANGELOG.md:1]: There is also a pub.dev standards which consider non pre-id based release numbers as "releases" and with pre-id or tagged releases as pre-releases. e.g. https://pub.dev/packages/artemis/versions

Comment (Ashish-Nanda) [reply packages/amplify_analytics_pinpoint/CHANGELOG.md:1]: I think from @jamesonwilliams point, this kind of versioning probably lowers developer confidence in adoption. So maybe when we are closer to almost stable, switching to 0.x.x will indicate that better. Something to consider though I dont have a strong opinion either way.

**Pull request merged**